src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1000,6 +1023,26 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                    call mpas_pool_get_field(scratchPool, 'btrvel_temp', btrvel_tempField)
                    btrvel_temp => btrvel_tempField % array
 
+                   ! Subtract tidal potential from ssh, if needed
+                   !   Subtract the tidal potential from the current and new subcycle ssh and store and a work arrays.
+                   !   Then point sshSubcycleCur and  ssh SubcycleNew to the work arrays so the tidal potential terms 
+                   !   are included in the grad operator inside the edge loop.
+                   if (config_use_tidal_potential_forcing) then
+                     call mpas_pool_get_array(forcingPool,'sshSubcycleCurWithTides', sshSubcycleCurWithTides)
+                     call mpas_pool_get_array(forcingPool,'sshSubcycleNewWithTides', sshSubcycleNewWithTides)
+                     call mpas_pool_get_array(forcingPool, 'tidalPotentialEta', tidalPotentialEta)
+                     call mpas_pool_get_dimension(block % dimensions, 'nCells', nCellsPtr)
+
+                     nCells = nCellsPtr
+                     do iCell = 1, nCells
+                       sshSubcycleCurWithTides(iCell) = sshSubcycleCur(iCell) - tidalPotentialEta(iCell)
+                       sshSubcycleNewWithTides(iCell) = sshSubcycleNew(iCell) - tidalPotentialEta(iCell)
```

Comment:
Where are `sshSubcycleCurWithTides` and `sshSubcycleNewWithTides` actually used in the code?  I can only find them being computed here but not used to update any prognostic variables.

Reply:
Sorry, I get it now.  I'm still getting tripped up by pulling in a variable with one name from a pool and using it with another name in the code, but I understand that that's practical here.